config/axios.js

```diff
@@ -0,0 +1,9 @@
+const axios = require('axios');
+
+const api = axios.create({
+  baseURL: process.env.EXTERNAL_API,
+  timeout: 3000,
+  headers: { 'Content-Type': 'application/json' }
```

Comment:
Generally headers should be handle per request, unless you are sure that you'll need them in every request. In this case you don't even need this header.